subclass/Shirayuri; Tierras Devastadas Bonded Eldritch Knight.json

```diff
@@ -0,0 +1,568 @@

```

Comment:
this'll get scraped off by the HTML remover; you might want to replace with proper `entries`